packages/protocol/releaseData/initializationData/release3.json

```diff
@@ -1,4 +1,4 @@
 {
-  "StableTokenEUR": ["Celo Euro", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", "0x0000000000000000000000000000000000000000", "0"],
+  "StableTokenEUR": ["Celo Euro", "cEUR", 18, "0x000000000000000000000000000000000000ce10", "1", "47304000", "0x0000000000000000000000000000000000000000", "0", "ExchangeEUR"],
```

Comment:
I'm assuming most of the parameters in this file are placeholders? Too bad json doesn't allow comments :/